todo_app/app.py

```diff
@@ -14,31 +14,29 @@ def create_app():
     
     @app.route('/')
     def index():
-        response_json = trello_items.get_trello_cards()
+        mongo_items = mongo_items_Vtwo.get_mongo_items()
 
         items = []
 
-        item_view_model = ViewModel(items)
-
-        for trello_list in response_json:
-            for card in trello_list['cards']:
-                item = Item.from_trello_card(card, trello_list)
+        for mongo_item in mongo_items: 
+                item = Item.from_mongo_item(mongo_item)
                 items.append(item)
+        item_view_model = ViewModel(items)
 
         return render_template('index.html', view_model=item_view_model)
     
     @app.route('/create-todo', methods=['Post'])
     def create_new_todo():
     
-        response = trello_items.create_todo()
+        response = mongo_items_Vtwo.create_mongo_todo_item()
         
         return index()
 
 
     @app.route('/update_status', methods=['POST'])
     def update_status():
         
-        response = trello_items.change_status()
+        response = mongo_items_Vtwo.change_mongo_status()
```

Comment:
The few changes you have had to make in this file is a good testament to putting sensible function definitions in place during the earlier modules - nice 😄